docs/src/main/paradox/mongodb.md

```diff
@@ -36,7 +38,38 @@ And finally we can run it.
 Scala
 : @@snip ($alpakka$/mongodb/src/test/scala/akka/stream/alpakka/mongodb/MongoSourceSpec.scala) { #run-source }
 
-Here we used a basic sink to complete the stream by collecting all of the stream elements to a collection. The power of streams comes from building larger data pipelines which leverage backpressure to ensure efficient flow control. Feel free to edit the example code and build @extref[more advanced stream topologies](akka-docs:scala/stream/stream-introduction).
+Here we used a basic sink to complete the stream by collecting all of the stream elements to a collection.
+
+### Sink Usage
```

Comment:
If you do not want to duplicate examples for flow vs sink I think We could call this "Flow and Sink Usage" and then add a paragraph directly below saying something like "Each of these sink factory methods have a corresponding factory in `MongoFlow` which will emit the written document or result of the operation downstream"

Reply:
Sounds good.

Reply:
Still missing this one, I think it is done for merging except for this.

Reply:
Ah sorry, thanks. All fixed up.